vms/platformvm/state/state.go

```diff
@@ -1133,7 +1158,7 @@ func (s *state) loadCurrentValidators() error {
 			return fmt.Errorf("expected tx type txs.Staker but got %T", tx.Unsigned)
 		}
 
-		staker, err := NewCurrentStaker(txID, stakerTx, metadata.PotentialReward)
+		staker, err := NewCurrentStaker(txID, stakerTx, stakerTx.StartTime(), metadata.PotentialReward)
```

Comment:
This is wrong: post fork stakers start time should not be stakerTx.StartTime() but it should be chainTime at staker creation, which then must be stored among  validator metadata